### DISCUSSION TITLE
courses/artificialintelligence/assignments/maze/

### DISCUSSION THREAD

**Discussion by giscus** [InfiniBrains/Awesome-GameDev-Resources#34] [General]
# courses/artificialintelligence/assignments/maze/

Awesome GameDev Resources for learning how to make games

https://courses.tolstenko.net/courses/artificialintelligence/assignments/maze/

**Comment by isaacHhyphenR** [comment]
The coordinates in the output section seem to be written in (Y,X) format rather than the traditional (X,Y). is that intentional?

**Reply by tolstenko** [reply]
Hi @isaacHhyphenR, which section? It might be 2 things:
- My dyslexia. If it is the reason, please hit the button on top right with a edit icon and fix it for me by submitting a pull-request :-) ;
- I am improving cache locality. Read more here: https://courses.tolstenko.net/courses/artificialintelligence/readings/spatial-quantization/#notes-on-cache-locality

**Reply by isaacHhyphenR** [reply]
It's in the 'Output' section walking through how the generation works.
